src/containers/SnippetArea.jsx

```diff
@@ -1,12 +1,13 @@
-import React, { PropTypes } from 'react';
-import { connect } from 'react-redux';
 import axios from 'axios';
-import _ from 'lodash'
+import _ from 'lodash';
 import { CardText, Snackbar } from 'material-ui';
-import { browserHistory } from 'react-router'
+import React, { PropTypes } from 'react';
```

Comment:
It's not hard to read a few lines down, and I've sorted the dependencies alphabetically. That's all